staging/cse/windows/provisioningscripts/kubeletstart.ps1

```diff
@@ -63,10 +63,6 @@ $KubeletArgListStr = "@($KubeletArgListStr`)"
 # Used in Azure-CNI version of kubeletstart.ps1
 $KubeletCommandLine = "$global:KubeDir\kubelet.exe " + ($KubeletArgList -join " ")
 
-# Turn off Firewall to enable pods to talk to service endpoints. (Kubelet should eventually do this)
-# TODO move this to CSE
-netsh advfirewall set allprofiles state off
```

Comment:
Can we leave this line here? Because the agentbaker changes are released seperately from this file, we might have a point in time where the firewall is not disabled in either place. And disabling it in both places seems better than not disabling it.